whipper/program/cdparanoia.py

```diff
@@ -205,8 +205,6 @@ def getTrackQuality(self):
 class ReadTrackTask(task.Task):
     """
     I am a task that reads a track using cdparanoia.
-
-    @ivar reads: how many reads were done to rip the track
```

Comment:
Is there a reason this is removed instead of replaced? Is it because it’s referring to something that’s been removed? I’m *slightly* leaning towards converting it and then removing it in #389. But eh.

Reply:
> Is it because it’s referring to something that’s been removed?

That's it.